pkg/kubeinteraction/labels.go

```diff
@@ -25,13 +31,36 @@ func AddLabelsAndAnnotations(event *info.Event, pipelineRun *tektonv1beta1.Pipel
 		filepath.Join(pipelinesascode.GroupName, "branch"):         formatting.K8LabelsCleanup(event.BaseBranch),
 		filepath.Join(pipelinesascode.GroupName, "repository"):     formatting.K8LabelsCleanup(repo.GetName()),
 		filepath.Join(pipelinesascode.GroupName, "git-provider"):   providerinfo.Name,
+		filepath.Join(pipelinesascode.GroupName, "state"):          StateStarted,
 	}
 
 	annotations := map[string]string{
 		filepath.Join(pipelinesascode.GroupName, "sha-title"): event.SHATitle,
 		filepath.Join(pipelinesascode.GroupName, "sha-url"):   event.SHAURL,
 	}
 
+	if event.PullRequestNumber != 0 {
+		annotations[filepath.Join(pipelinesascode.GroupName, "pull-request")] = strconv.Itoa(event.PullRequestNumber)
+	}
+
+	// TODO: move to provider specific function
+	if providerinfo.Name == "github" || providerinfo.Name == "github-enterprise" {
+		if event.InstallationID != -1 {
+			annotations[filepath.Join(pipelinesascode.GroupName, "installation-id")] = strconv.FormatInt(event.InstallationID, 10)
+		}
+		if event.GHEURL != "" {
+			annotations[filepath.Join(pipelinesascode.GroupName, "ghe-url")] = event.GHEURL
```

Comment:
could we not just grab this url from the repo crd instead of storing them in annotations

Reply:
ah yeah :)